OptimizelySDK/OptimizelyTests/OptimizelyTests-Common/LoggerTests.swift

```diff
@@ -38,6 +38,23 @@ class LoggerTests: XCTestCase {
     }
 }
 
+// MARK: - DefaultLogger Tests
+
+extension LoggerTests {
```

Comment:
Please add this as a test and not an extension.  Why do you have it as an extension?